information/mqtt_topics_and_message_structure.md

```diff
@@ -191,19 +192,54 @@ Execute selftest
 
 ### Xiaomi Aqara curtain motor (ZNCLDJ11LM)
 Set the state of the curtain.
-```
+```js
 {
   "state": "open"       // Possible values to set: 'open', 'close', 'stop'
 }
 ```
 
 Set the position of the curtain.
-```
+```js
 {
   "position": 50      // Possible values to set: 0 - 100 (0 - closed / 100 - open)
 }
 ```
 
+### Xiaomi Aqara wired wall switch (QBKG03LM and QBKG04LM)
+Decoupled mode allows to turn wired switch into wireless button with separately controlled relay.
+This might be useful to assign some custom actions to buttons and control relay remotely.
+This command also allows to redefine which button controls which relay for double switch.
+
+Special topics should be used:
+
+`zigbee2mqtt/[DEVICE_ID]/system/set` to modify operating mode.
+
+Payload:
+```js
+{
+  "decoupled_mode": {
+    "button": "single"|"left"|"right",
+    "state": "decoupled"|"toggle_left"|"toggle_right"
+  }
+}
+```
+`zigbee2mqtt/[DEVICE_ID]/system/get` to read current mode.
+
+Payload:
+```js
+{
+  "decoupled_mode": {
+    "button": "single"|"left"|"right"
+  }
+}
+```
+
+Response      | Description
+--------------|-----------------------------------------------
+18            | Button controls left relay
+34            | Button controls right relay
+254           | Button is decoupled from the relays
```

Comment:
Instead of returning numeric codes, why is this converter not just returning text like: `control_left_relay`, `decoupled`? That would avoid users having to lookup the meaning of the code.

Reply:
Because this converter only sends query. To parse this, new converter should be added to read the response. But yeah, I can update this.

Reply:
I don't understand, where is this response is currently visible?

Reply:
Right now only in debug logs of zigbee2mqtt.

Reply:
With a 'No converter available..' message?

Reply:
No, it has ignore converter.
I updated it in https://github.com/Koenkk/zigbee-shepherd-converters/pull/290.